Berlin/BerOrOct1270.xml

```diff
@@ -188,21 +188,30 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <decoDesc>
 <decoNote type="miniature" xml:id="d1">
 <locus target="#1r"></locus>
+<desc><ref type="authFile" corresp="AT1035HolyMan"></ref></desc>
+<q n="1" xml:lang="gez">ስዕለ፡ ቅዱስ፡ <gap reason="illegible"></gap>ርማም።</q>
+<note>The caption is only partly readable. The ink colour looks red.</note>
 </decoNote>
 <decoNote type="miniature" xml:id="d2">
 <locus target="#1v"></locus>
+<desc><ref type="authFile" corresp="AT1035HolyMan">Gabra Krǝstos</ref></desc>
```

Comment:
I think it would be clearer, and result in better visualisation, to encode this as `A <ref type="authFile" corresp="AT1035HolyMan"/> of Gabra Krǝstos`. Of course, it would also be possible to encode the names of the protagonists here and below, if you want to invest that time.

Reply:
I have opted for the simplest solution.. No time to invest for now :-),
Thanks both.